store.go

```diff
@@ -1109,6 +1110,14 @@ func (s *store) getROLayerStoresLocked() ([]roLayerStore, error) {
 		return nil, err
 	}
 	for _, store := range s.graphDriver.AdditionalImageStores() {
+		if strings.Contains(store, "|") {
+			parts := strings.Split(store, "|")
+			if len(parts) > 1 && parts[1] == "NO_REUSE" {
```

Comment:
We can invent an infinite number of new options. Please don’t add ambiguous in-line syntaxes unless there is really no other alternative.